rotkehlchen/db/schema.py

```diff
@@ -244,6 +244,36 @@
 );
 """
 
+DB_CREATE_YEARN_V2_VAULT_EVENTS = """
+CREATE TABLE IF NOT EXISTS yearn_v2_vaults_events (
+    address VARCHAR[42] NOT NULL,
+    event_type VARCHAR[10] NOT NULL,
+    from_asset VARCHAR[44] NOT NULL,
+    from_amount TEXT NOT NULL,
+    from_usd_value TEXT NOT NULL,
+    to_asset VARCHAR[44] NOT NULL,
+    to_amount TEXT NOT NULL,
+    to_usd_value TEXT NOT NULL,
+    pnl_amount TEXT,
+    pnl_usd_value TEXT,
+    block_number INTEGER NOT NULL,
+    timestamp INTEGER NOT NULL,
+    tx_hash VARCHAR[66] NOT NULL,
+    log_index INTEGER NOT NULL,
+    PRIMARY KEY (event_type, tx_hash, log_index)
+);
```

Comment:
Why create a different DB table? I don't see any difference to the v1 table.

Reply:
solved by adding a version column